### PR TITLE
<TimeInput/> - refactor story page

### PR DESCRIPTION
**Pull request by israelabuhay**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6158

File: src/TimeInput/docs/index.story.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+  api,
+  testkit,
+} from 'wix-storybook-utils/Sections';
 import TimeInput from '..';
 import { storySettings } from './storySettings';
+import allComponents from '../../../stories/utils/allComponents';
 import LockLocked from 'wix-ui-icons-common/LockLocked';
 import Input from '../../Input';
+import * as examples from './examples';
+import { Cell, Layout } from 'wix-style-react';
```
Comment (gendeld) [inline src/TimeInput/docs/index.story.js:21]: It's better to import components relatively:
```
import { Layout, Cell } from '../../Layout';
```

File: src/TimeInput/docs/index.story.js [hunk 2]
```diff
@@ -33,4 +51,61 @@ export default {
       },
     ],
   },
+  sections: [
+    header({
+      sourceUrl: `https://github.com/wix/wix-style-react/tree/master/src/${TimeInput.displayName}/`,
+    }),
```
Comment (gendeld) [inline src/TimeInput/docs/index.story.js:57]: I think you can just do
`header(),`
`sourceUrl` should be derived automatically.

File: src/TimeInput/docs/index.story.js [hunk 3]
```diff
@@ -33,4 +51,61 @@ export default {
...
+          example({
+            title: 'With status',
+            text: 'TimeInput supports `error`, `warning`, and `loading` status',
+            source: examples.status,
+          }),
+          example({
+            title: 'With suffix',
+            text: 'TimeInput supports `customSuffix` to display before ticker',
+            source: examples.customSuffix,
+            components: { TimeInput, Input, LockLocked, Layout, Cell },
```
Comment (gendeld) [inline src/TimeInput/docs/index.story.js:94]: I'm not sure we need this line

**Comment by gendeld** [comment]
What's here is excellent 🎖,
we can beef up the `Playground` and `Testkit` tabs.

I think we should add `disableAmPm` and `status` to the Playground tab.
Playground is automatically created from the exported story:
`componentProps` - controls which props to show and what the default value is,
the value of each prop is derived from component's propType,
`exampleProps` - is for complicated props where we want to give a few preset examples.
You can look at the `InputArea` story to see how they added `status`.

In the Testkit tab,
we want the exposed methods to have descriptions.
The descriptions are automatically derived from the comments above each method in a `.uni.driver.js` file.
You can look at `src/ToggleButton/ToggleButton.uni.driver.js` to see how they document each method with a return value as well (you can check `src/TimeInput/TimeInput.uni.driver.d.ts` for each methods return value).

Keep up the great work 🙌

File: src/TimeInput/docs/index.story.js (hunk 3 above)
Comment (israelabuhay) [reply src/TimeInput/docs/index.story.js:108]: It for the `customSuffix` with the icon will work properly

File: src/TimeInput/TimeInput.uni.driver.js [hunk 4]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
     base.$(`[data-hook="${dataHooks.amPmIndicator}"]`);
   const input = testkit(base.$(`[data-hook="${dataHooks.input}"]`), body);
   const inputTicker = tickerDriverFactory(base);
+  const timeInputDriver = baseUniDriverFactory(base);
 
   return {
-    ...baseUniDriverFactory(base),
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:13]: We do want the base methods to be inherited from `baseUniDriverFactory`.
The comments are added to the base methods automatically.

File: src/TimeInput/TimeInput.uni.driver.js [hunk 5]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
...
+     * @returns {Promise<void>}
+     */
     clickTickerUp: async () => inputTicker.clickUp(),
+    /**
+     * Clicks on TimeInput down ticker
+     * @returns {Promise<void>}
+     */
     clickTickerDown: async () => inputTicker.clickDown(),
+    /**
+     * Checks whether TimeInput 24h mode exist
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:50]: It checks whether the AM/PM indicator/toggle exists

File: src/TimeInput/TimeInput.uni.driver.js [hunk 6]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
...
+     * @returns {Promise<boolean>}
+     */
     isAmPmIndicatorExist: async () => amPmIndicator().exists(),
+    /**
+     * Clicks on 24h mode toggle
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:55]: It toggles AM/PM indicator (doesn't switch modes)

File: src/TimeInput/TimeInput.uni.driver.js [hunk 7]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
...
+     */
     toggleAmPmIndicator: async () => amPmIndicator().click(),
+    /**
+     * Gets the amPm indicator text
+     * @returns {Promise<string>}
+     */
     getAmPmIndicatorText: async () => amPmIndicator().text(),
+    /**
+     * Gets the custom suffix
+     * @returns {Promise<React.ReactNode>}
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:66]: This is a mistake in the type declaration,
_prop('innerHTML') should return Promise<string>

File: src/TimeInput/TimeInput.uni.driver.js (hunk 4 above)
Comment (israelabuhay) [reply src/TimeInput/TimeInput.uni.driver.js:13]: The comments are not added.

File: src/TimeInput/TimeInput.uni.driver.js [hunk 8]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
...
+     * @returns {Promise<void>}
+     */
     setValue: async value => input.enterText(value),
+    /**
+     * Trigger blur on the TimeInput
+     * @returns {Promise<void>}
+     */
     blur: async () => input.blur(),
+    /**
+     * Checks if TimeInput has status
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:86]: add
`@param {string} status status to check`

File: src/TimeInput/TimeInput.uni.driver.js [hunk 9]
```diff
@@ -8,22 +8,89 @@ export const timeInputUniDriverFactory = (base, body) => {
...
+     */
     getCustomSuffix: () =>
       base.$(`[data-hook="${dataHooks.customSuffix}"]`)._prop('innerHTML'),
+    /**
+     * Checks if TimeInput is RTL
+     * @returns {Promise<boolean>}
+     */
     isRtl: async () => input.isRTL(),
+    /**
+     * Set value to display in the TimeInput
```
Comment (gendeld) [inline src/TimeInput/TimeInput.uni.driver.js:76]: add
`@param {string} value text value to input`

**Review by gendeld — APPROVED** [review]

**Pull request merged**